src/test/java/org/jabref/logic/layout/format/HTMLCharsTest.java

```diff
@@ -81,7 +77,7 @@ private static Stream<Arguments> provideEquationTestData() {
         return Stream.of(
                 Arguments.of("&dollar;", "\\$"),
                 Arguments.of("&sigma;", "$\\sigma$"),
-                Arguments.of("A 32&nbsp;mA &Sigma;&Delta;-modulator", "A 32~{mA} {$\\Sigma\\Delta$}-modulator")
+                Arguments.of("A 32&nbsp;mA &Sigma;&Delta;-modulator", "A 32~mA $\\Sigma\\Delta$-modulator")
```

Comment:
The code change is OK. Reason: `{` is not used as parameter for a LaTeX command.